ethereum/spec/src/test/java/tech/pegasys/teku/spec/datastructures/networking/libp2p/rpc/BlobSidecarsByRootRequestMessageTest.java

```diff
@@ -58,7 +58,10 @@ public void verifyMaxLengthOfContainerIsGreaterOrEqualToMaxRequestBlobSidecars()
           final UInt64 maxRequestBlobSidecars =
               SpecConfigDeneb.required(spec.forMilestone(milestone).getConfig())
                   .getMaxRequestBlobSidecars();
-          assertThat(BeaconBlocksByRootRequestMessage.SSZ_SCHEMA.getMaxLength())
+          assertThat(
+                  spec.getGenesisSchemaDefinitions()
+                      .getBeaconBlocksByRootRequestMessageSchema()
+                      .getMaxLength())
```

Comment:
everything else in the entire PR is coming from Network configuration, any reason this needs to be different? because this is kind of ugly now...

Reply:
you mean to get max request blocks from networking constants instead of schema? Makes sense